### PR TITLE
execute tcp liveness probe instance

### PR DESCRIPTION
**Pull request by chenhonggc**
Execute tcp liveness probe instance.

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md [hunk 1]
```diff
@@ -195,6 +195,18 @@ starts. Just like the readiness probe, this will attempt to connect to the
 `goproxy` container on port 8080. If the liveness probe fails, the container
 will be restarted.
 
+To try the TCP liveness check, create a Pod:
+
+```shell
+kubectl create -f https://k8s.io/docs/tasks/configure-pod-container/http-liveness.yaml
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md]: You should use the `tcp-liveness-readiness.yaml` file for TCP probes.

**Comment by chenhonggc** [comment]
@tengqm Sorry, I was careless， Please review again.

**Comment by chenhonggc** [comment]
@steveperry-53 Please approve thanks.

**Pull request merged**